src/main/java/nl/hsac/fitnesse/fixture/slim/mobile/AndroidAppTest.java

```diff
@@ -0,0 +1,221 @@
+package nl.hsac.fitnesse.fixture.slim.mobile;
+
+import io.appium.java_client.*;
+
+import io.appium.java_client.android.AndroidDriver;
+import nl.hsac.fitnesse.fixture.slim.SlimFixtureException;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import io.appium.java_client.remote.MobileCapabilityType;
+import io.appium.java_client.remote.MobilePlatform;
+
+import nl.hsac.fitnesse.fixture.slim.web.BrowserTest;
+import nl.hsac.fitnesse.fixture.slim.web.annotation.TimeoutPolicy;
+import nl.hsac.fitnesse.fixture.slim.web.annotation.WaitUntil;
+
+import java.lang.reflect.Method;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Android App fixture for use with an android driver. Aimed at effortlessly testing native apps
+ */
+
+public class AndroidAppTest extends BrowserTest {
+    private int SWIPESPEED = 1000;
+    private int WAITAFTERSWIPE = 0;
+
+    public AndroidAppTest() {
+        super("X");
+    }
+
+    @Override
+    protected void beforeInvoke(Method method, Object[] arguments) {
+        //intentionally left empty..
```

Comment:
Why is it left empty. Just to ensure no waiting for angular?
It would be easier/clearer to, in the constructor just call `setImplicitWaitForAngularTo(false)`